nixos/modules/services/security/aide.nix

```diff
@@ -0,0 +1,145 @@
+{ config, lib, pkgs, ... }:
+with lib;
+let
+  cfg = config.services.aide;
+  configFile = pkgs.writeText "aide.conf" ''
+    # The location of the database
+    database=${cfg.database_url}
```

Comment:
has been changed to `database_in` in v0.17, `database` will deprecate in AIDE v0.19
https://github.com/aide/aide/blob/v0.18.x/doc/aide.conf.5#L34-L36